include/bluetooth/l2cap.h

```diff
@@ -191,10 +180,23 @@ struct bt_l2cap_le_chan {
 	struct k_work			tx_work;
 	/** Segment SDU packet from upper layer */
 	struct net_buf			*_sdu;
-	uint16_t				_sdu_len;
+	uint16_t			_sdu_len;
 
 	struct k_work			rx_work;
 	struct k_fifo			rx_queue;
+
+#if defined(CONFIG_BT_L2CAP_DYNAMIC_CHANNEL)
+	bt_l2cap_chan_state_t		state;
+	/** Remote PSM to be connected */
+	uint16_t			psm;
+	/** Helps match request context during CoC */
+	uint8_t				ident;
+	bt_security_t			required_sec_level;
+
+	/* Response Timeout eXpired (RTX) timer */
+	struct k_work_delayable		rtx_work;
+	struct k_work_sync              rtx_sync;
```

Comment:
While the code style indicate that indentation should be done with space after non-whitespace characters, perhaps it's better to use the same style of indention, instead of using tabs for some of the field, and spaces for others.

Reply:
Yes, structs should be consistently formatted. It would be good to go for cross-subsystem consistency with using only spaces for member name alignment, but that needs to be done for entire structs (and even entire c-files) at a time, instead of only for individual struct members.